ui/app/src/views/auth/SignWrapper.tsx

```diff
@@ -52,22 +150,26 @@ export function SignWrapper(props: { children: ReactNode }) {
       />
       <Stack gap={1} sx={{ maxWidth: '85%', minWidth: '200px' }}>
         {nativeProviderIsEnabled && props.children}
-        {socialProviders.length > 0 && (
+        {nativeProviderIsEnabled && socialProviders.length > 0 && (
           <div>
-            <Divider sx={{ marginTop: '16px' }}>{nativeProviderIsEnabled && 'or'} sign in with</Divider>
+            <Divider sx={{ marginTop: '16px' }}>{nativeProviderIsEnabled && 'or'}</Divider>
```

Comment:
```suggestion
            <Divider sx={{ marginTop: '16px' }}>or</Divider>
```

Reply:
The idea is to have the  'or' only if the nativeProvider is enabled. Can we talk about what you mean ?

Reply:
aaah okay I understood, let me try